python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -809,6 +789,28 @@ def date_range(
                 '2026-04-23 08:00:00'],
                 dtype='datetime64[ns]')
 
+    .. pandas-compat::
+        **cudf.date_range**
+
+        Of the four parameters `start`, `end`, `periods`, and `freq`, exactly three
+        must be specified. If `freq` is omitted, the resulting DatetimeIndex will
+        have periods linearly spaced elements between start and end (closed on both
+        sides).
```

Comment:
This part is not a pandas compatibility note, so please restore it to its previous location.